app/bot/__init__.py

```diff
@@ -11,7 +11,35 @@
 _intents.message_content = True
 
 client = discord.Client(intents=_intents)
+conversation_history = []
 
+# FIXME: This will probably have race condition issues, GL HF :)
+def update_conversation_history(user_message, bot_response):
```

Comment:
All functions related to the LLM should be in the [`llm` module](https://github.com/btc-raspberrypiclub/piclub-bot/blob/main/app/bot/llm.py)

Reply:
Makes sense, will fix